crypto/openssl/openssl_key_exchange_ecdh.c

```diff
@@ -245,24 +250,27 @@ static bool compute_publicBECDH(ecdh_context_t *key_ex_data)
 	}
 
 	tmp = bn_num_bytes(x);
-	if (tmp & 0xffff0000) // check size more than 2 byte size space
+	if (tmp & 0xffff0000) {// check size more than 2 byte size space
 		goto exit;
+	}
 	temp[0] = tmp >> 8;
 	size = 1;
 	temp[size] = (tmp & 0x00ff);
 	size += 1;
 	size += BN_bn2bin(x, &temp[size]);
 	tmp = bn_num_bytes(y);
-	if (tmp & 0xffff0000) // check size more than 2 byte size space
+	if (tmp & 0xffff0000) {// check size more than 2 byte size space
 		goto exit;
+	}
 	temp[size] = tmp >> 8;
 	size += 1;
 	temp[size] = (tmp & 0x00ff);
 	size += 1;
 	size += BN_bn2bin(y, &temp[size]);
 	tmp = bn_num_bytes(key_ex_data->_Device_random);
-	if (tmp & 0xffff0000) // check size more than 2 byte size space
+	if (tmp & 0xffff0000) {// check size more than 2 byte size space
```

Comment:
nitpick - should there be a space between the brace and comment (or comment on separate line)